src/main/java/creativei/helper/ResponseHelper.java

```diff
@@ -73,8 +70,16 @@ public static FollowUpVo getCreateFollowUpData(FollowUp followUp, FollowUpVo fol
         followUpVo.setSubStatus(FollowUpSubStatus.enumToString(followUp.getSubStatus()));
         return followUpVo;
     }
-  public static ReminderVo getCreateReminderResponse(Reminder reminder){
-        ReminderVo reminderVo=new ReminderVo(reminder);
-        return reminderVo;
+
+    public static ReminderVo getCreateReminderResponse(Reminder reminder) {
+        return new ReminderVo(reminder);
+    }
+
```

Comment:
This method is common for many methods. Create a private method and use that

Reply:
sir don't get this which method is common?

Reply:
Method where you initialize arraylist and iterate to add Inquiry objects.

Reply:
added in commit 6eb1ebb